ivy/functional/frontends/numpy/linear_algebra/linalg/solving_equations_and_inverting_matrices.py

```diff
@@ -0,0 +1,20 @@
+# global
+import ivy
+
+# inv
+
+
+def inv(a):
+    return ivy.inv(a)
+
+
+inv.unsupported_dtypes = {"torch": ("float16",)}
```

Comment:
you don't have to specify `torch` here, this is for unsupported dtypes of the frontend function, and since `numpy.linalg` doesn't support `float16`, you can just simlpy add 
`inv.unsupported_dtypes = ("float16",)`